src/Configuration/Definition/BaseExtractorConfigDefinition.php

```diff
@@ -88,6 +88,7 @@ protected function getSshNode(): ArrayNodeDefinition
                         ->scalarNode('private')->end()
                         ->scalarNode('#private')
                             ->isRequired()
+                            ->cannotBeEmpty()
                         ->end()
                         ->scalarNode('public')->end()
```

Comment:
should public key also be required?

Reply:
I'm not really sure about it. The public key is never used in component.